libs/shared/ui/src/components/avatar-modal/AvatarModal.styled.tsx

```diff
@@ -0,0 +1,54 @@
+import { Color, space } from '@mentorship/shared/variables';
+import styled from 'styled-components';
+
+import { CardButtonWrapper } from '../card/Card.styled';
+import { ImageBody } from '../image/Image.styled';
+
+export const AvatarModalWrapper = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  display: grid;
+  place-items: center;
+  background: ${Color.white};
+  padding: ${space.x8};
+  border: 2px solid ${Color.dodgerblue};
+  border-radius: 5px;
+  z-index: 2;
+`;
+
+export const AvatarTitle = styled.h1``;
+
+export const AvatarBody = styled.div`
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: ${space.x3};
+  margin: ${space.x4} 0;
+`;
+
+export const AvatarImage = styled(ImageBody)<{ $isSelected: boolean }>`
+  position: static;
+  border: ${({ $isSelected }) => ($isSelected ? `4px solid ${Color.dodgerblue}` : `2px solid ${Color.gray}`)};
+  transition: 0.35s ease-in-out;
+
+  &:hover {
+    transform: scale(1.6);
+    cursor: pointer;
+  }
+`;
+
+export const AvatarButtonWrapper = styled(CardButtonWrapper)``;
```

Comment:
Same as comment above ☝️

Reply:
Very nice catch. I forgot to remove unnecessary styled components when I was creating a reusable Modal. There are also a few more components that need to be removed.